Ginger/Ginger/ALM/MapToALMWizard/TestStepMappingPage.xaml.cs

```diff
@@ -144,7 +144,7 @@ private void UnmapAllRows(object sender, RoutedEventArgs e)
             }
             catch (Exception ex)
             {
-                Reporter.ToLog(eLogLevel.DEBUG, ex.Message);
+                Reporter.ToLog(eLogLevel.DEBUG, ex.ToString());
```

Comment:
change it to Reporter.ToLog(eLogLevel.DEBUG, "<Error related msg>", ex);

Reply:
Done